go-controller/pkg/ovn/policy_test.go

```diff
@@ -1234,25 +1227,29 @@ var _ = Describe("OVN NetworkPolicy Operations", func() {
 						},
 					},
 				)
+				fakeOvn.controller.WatchNamespaces()
 				nPodTest.populateLogicalSwitchCache(fakeOvn)
 				fakeOvn.controller.WatchPods()
-				fakeOvn.controller.WatchNamespaces()
```

Comment:
this indicates there is something else wrong with the patch here. watch namespace handler will handle pods that did not get hit due to the watch ordering for pods being first.

Reply:
The problem with this ordering is that the test itself misses the namespace events. When i run it locally in this order it passes, but not upstream, so some timing issue here. And seems like this patch is kind of surfacing those issue. Given that i wait for 5 second now before matching the command, ig might work with the ordering, so let me revert and see.

Reply:
But does this ordering matters? because it anyways enable multicast, once pod is part of the namespace, so not sure calling WatchNamespaces before WatchPods changes anything in term of the test?

Reply:
@trozet this is the issue 

```
I1014 20:11:54.568942   14616 reflector.go:175] Starting reflector *v1.EgressIP (0s) from pkg/crd/egressip/v1/apis/informers/externalversions/factory.go:117
I1014 20:11:54.568960   14616 reflector.go:211] Listing and watching *v1.EgressIP from pkg/crd/egressip/v1/apis/informers/externalversions/factory.go:117
I1014 20:11:54.668984   14616 shared_informer.go:253] caches populated
I1014 20:11:54.669298   14616 kube.go:63] Setting annotations map[k8s.ovn.org/pod-networks:{"default":{"ip_addresses":["10.128.1.3/24"],"mac_address":"0a:58:0a:80:01:03","gateway_ips":["10.128.1.1"],"ip_address":"10.128.1.3/24","gateway_ip":"10.128.1.1"}}] on pod namespace1/myPod
I1014 20:12:04.670016   14616 pods.go:259] [namespace1/myPod] addLogicalPort took 10.000733062s
**E1014 20:12:04.670049   14616 ovn.go:546] timeout waiting for namespace event <<<<<<<<<<<HERE**
I1014 20:12:04.670083   14616 ovn.go:591] Bootstrapping existing pods and cleaning stale pods took 10.000911062s
I1014 20:12:04.670154   14616 ovn.go:875] Bootstrapping existing namespaces and cleaning stale namespaces took 63.7µs
I1014 20:12:04.670295   14616 pods.go:295] LSP already exists for port: namespace1_myPod
```